elliott

```diff
@@ -1038,7 +1038,143 @@ def find_cve_trackers(runtime, cve, status):
+    if rc != 0:
+        # Probably no point in continuing.. can't contact brew?
+        print("Unable to run oc release info: out={}  ; err={}".format(stdout, stderr))
+        exit(1)
+    else:
+        click.echo("Got release info")
+
+    payload_json = json.loads(stdout)
+
+    green_prefix("Looping over payload images: ")
+    click.echo("{} images to check".format(len(payload_json['references']['spec']['tags'])))
+    for image in payload_json['references']['spec']['tags']:
+        click.echo("----")
+        green_prefix("Getting payload image metadata: ")
+        click.echo("{}".format(image['from']['name']))
+        pullspec = image['from']['name']
+        image_name = image['name']
+        pullspec_cmd = 'oc image info {} -o json'.format(pullspec)
+        rc, stdout, stderr = exectools.cmd_gather(pullspec_cmd)
+        if rc != 0:
+            # Probably no point in continuing.. can't contact brew?
+            red_prefix("Unable to run oc image info: ")
+            click.echo("out={}  ; err={}".format(stdout, stderr))
+            exit(1)
+
+        image_info = json.loads(stdout)
+        labels = image_info['config']['config']['Labels']
+
+        # The machine-os-content image doesn't follow the standard
+        # pattern. We need to skip that image when we find it, it is
+        # not attached to advisories.
+        if 'com.coreos.ostree-commit' in labels:
+            yellow_prefix("Skipping machine-os-content image: ")
+            click.echo("Not required for checks")
+            continue
+
+        component = labels['com.redhat.component']
+        n = image_name
+        click.echo("Payload name: {}".format(n))
+        click.echo("Brew name: {}".format(component))
+        if labels:
+            v = labels['version']
+            r = labels['release']
+            all_payload_nvrs[component] = "{}-{}".format(v, r)
+        else:
+            print("For image {} Labels doesn't exist, image_info: {}".format(image_name, image_info))
+
+    missing_in_errata = {}
+    payload_doesnt_match_errata = {}
+    output = {
+        'missing_in_advisory': missing_in_errata,
+        'payload_advisory_mismatch': payload_doesnt_match_errata,
+    }
+    green_prefix("Analyzing data: ")
+    click.echo("{} images to consider from payload".format(len(all_payload_nvrs)))
+
+    for image, vr in all_payload_nvrs.items():
+        yellow_prefix("Cross-checking from payload: ")
+        click.echo("{}-{}".format(image, vr))
+        if image not in all_advisory_nvrs:
+            missing_in_errata[image] = "{}-{}".format(image, vr)
+            click.echo("{} in payload not found in advisory".format("{}-{}".format(image, vr)))
+        elif image in all_advisory_nvrs and vr != all_advisory_nvrs[image]:
+            click.echo("{} from payload has version {} which does not match {} from advisory".format(
+                image, vr, all_advisory_nvrs[image]))
+            if vr > all_advisory_nvrs[image]:
```

Comment:
string comparison of 'v4.1.10' vs 'v4.1.9' will be incorrect.
but rather than get this right, you might consider omitting it. it's kind of clutter and the user could visually determine this themselves if the previous bit were formatted something like
```
ose-ovn-kubernetes-container version mismatch:
    payload  v4.1.0-201905231545
    advisory v4.1.0-201905191700 
```

Reply:
good catch.